crates/compiler/builtins/bitcode/src/fuzz_sort.zig

```diff
@@ -0,0 +1,101 @@
+const std = @import("std");
+const sort = @import("sort.zig");
+
+extern fn malloc(size: usize) callconv(.C) ?*anyopaque;
+extern fn free(c_ptr: *anyopaque) callconv(.C) void;
+
+fn cMain() callconv(.C) i32 {
+    fuzz_main() catch unreachable;
+    return 0;
+}
+
+comptime {
+    @export(cMain, .{ .name = "main", .linkage = .Strong });
+}
```

Comment:
Why do we do this? I'm guessing it's to force zig to keep the `main` symbol in the static library.

Reply:
Exactly that. And we have to build a static library cause Zig doesn't have the capability to generate an executable for fuzzing. Instead, we generate a static library and have c++ deal with instrumentation of the llvm ir and final linking